ecosystem/sep-0031.md

```diff
@@ -65,21 +65,24 @@ This protocol involves the transfer of value, and so HTTPS is required for all e
 ### User Onboarding
 
 1. User onboarding happens with the sending anchor, and is out of the scope of this spec.
-1. Sending anchor should use this to collect any KYC about the receiver it will need so it doesn't have to do it at every transaction.
+2. The sending anchor should, at minimum, collect the KYC information of the sending and receiving users specified by the receiving anchor's [SEP-12](sep-0012.md) implementation.
 
 ### Payment Flow
 
 1. Payment initiation happens in the sending anchors interface, and is also out of the scope of this spec.
 1. The sending client chooses a destination region.
 1. The sending anchor fetches the TOML file to find the DIRECT_PAYMENT_SERVER API root and [`/info`](#info) endpoint fields for the receiving anchor for this region.
-1. Using the fields fetched from /info, the sending anchor collects all needed information from the sending client.
-1. The sending anchor POSTs all that information to the [/send](#send) endpoint of the receiving anchor.
-1. If information is missing, or the receiving anchor needs extra information to complete this transaction, `POST /send` will return `400 Customer Information Needed` along with the missing fields. The sending anchor should collect these and retry the entire `POST /send` call, until it succeeds.
+1. For every `*_sep12_type` value specified in the relevant asset's [`/info`](#info) object, the sending anchor makes a [`SEP-12 GET /customer`](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0012.md#customer-get) request.
```

Comment:
we should add some small blurb about how the _type works, that its an arbitrary key that the anchor can use in order to inform the SEP12 endpoint what set of fields we need.  We can link to the sep12 section on it as well once that lands.

Reply:
done